src/runtime/validator.ts

```diff
@@ -33,34 +19,32 @@ export const useChecker = (
         html = format(html, { parser: 'html' })
         couldFormat = true
       }
-    // eslint-disable-next-line
-  } catch (e) {
-      reporter.error(e)
+    } catch (e) {
+      console.error(e)
     }
 
     // Clean up Vue scoped style attributes
     html = typeof html === 'string' ? html.replace(/ ?data-v-[a-z0-9]+\b/g, '') : html
-
     const { valid, results } = validator.validateString(html)
 
-    if (valid) {
-      return reporter.success(
-      `No HTML validation errors found for ${chalk.bold(url)}`
+    if (valid && !results.length) {
+      return console.log(
```

Comment:
Maybe use `consola` here?

Reply:
I saw that `consola` was not updated in quite some time. Is it still meant to be used or is there a successor? :)

Reply:
I saw that the `@nuxt/kit` implementation [still uses `consola`](https://github.com/nuxt/framework/blob/40d090745bfe4a816dbf9f587d210c3af32e47a9/packages/kit/src/logger.ts#L1), so consider my question answered :rocket:

Reply:
No, I'm deliberately keeping this code free of `@nuxt/kit` dependencies. Although it's never meant to be included in the build, I don't think we need to process the entirety of nuxt/kit in order to log an error.